### PR TITLE
add the Update function to update a user

### PR DESCRIPTION
**Pull request by luddite0**

### CODE REVIEW

**Comment by ocxo** [comment]
Could you please add a test to cover this?

**Comment by luddite0** [comment]
will do

**Comment by luddite0** [comment]
test added. a big TODO is to go back & dry up the test files.

File: okta/user_test.go [hunk 1]
```diff
@@ -151,6 +152,50 @@ func TestUserGet(t *testing.T) {
...
+		t.Errorf("Users.Update json UnMarshall returned error: %v", err)
+	}
+	testuser.Profile.FirstName = "Herschel"
+	temp, err := json.Marshal(testuser)
+	if err != nil {
+		t.Errorf("Users.Update json Marshall returned error: %v", err)
+	}
+	updateTestJSONString := string(temp)
+
+	mux.HandleFunc("/users/00ub0oNGTSWTBKOLGLNR", func(w http.ResponseWriter, r *http.Request) {
```
Comment (tecnobrat) [inline okta/user_test.go]: Where does the string `00ub0oNGTSWTBKOLGLNR` come from?

**Pull request merged**